pkg/test/framework/components/echo/common/deployment/echos.go

```diff
@@ -75,8 +75,8 @@ func (c *Config) fillDefaults(ctx resource.Context) error {
 	}
 
 	// Verify the namespace for any custom deployments.
-	if c.Custom != nil {
-		for _, custom := range c.Custom.Get() {
+	if c.Configs != nil {
```

Comment:
Actually, can we set c.Configs to the default value here if it's nil? Seems we'd kill two birds with one stone. Then you wouldn't need to check nil here.

Reply:
added one change, hopefully that resolves this query.